x/oracle/types/genesis.pb.go

```diff
@@ -142,6 +141,72 @@ func (m *QuotePrice) GetBlockHeight() uint64 {
 	return 0
 }
 
+// CurrencyPairState represents the stateful information tracked by the x/oracle
+// module per-currency-pair.
+type CurrencyPairState struct {
+	// QuotePrice is the latest price for a currency-pair, notice this value can
+	// be null in the case that no price exists for the currency-pair
+	Price *QuotePrice `protobuf:"bytes,1,opt,name=price,proto3" json:"price,omitempty"`
+	// Nonce is the number of updates this currency-pair has received
+	Nonce uint64 `protobuf:"varint,2,opt,name=nonce,proto3" json:"nonce,omitempty"`
+	// ID is the ID of the CurrencyPair
+	Id uint64 `protobuf:"varint,3,opt,name=id,proto3" json:"id,omitempty"`
+}
```

Comment:
Currency Pair State: info tracked in the state ( price - could be nil, if there were no price updates for the currency pair; nonce is the num of updates (so if price = nil -> nonce = 0); Id of currency pair in the oracle.